src/test/java/org/mitre/synthea/world/agents/PayerTest.java

```diff
@@ -25,29 +28,27 @@
 
 public class PayerTest {
 
+  static String testState;
```

Comment:
I know it's just a test and that it was this way previously, but default visibility is rarely used in Java.

Reply:
Fixed.